dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -9833,4979 +9833,4982 @@ BEGIN
+    11150 "Săgeată spre stânga în formă de u cu cap triunghiular în sens invers acelor de ceasornic"
+    11151 "Săgeată în sus în formă de u cu cap triunghiular în sens invers acelor de ceasornic"
+    11152 "Înapoi în stânga"
+    11153 "Înapoi în dreapta"
+    11154 "Rând nou în stânga"
+    11155 "Rând nou în dreapta"
+    11156 "Săgeți cu patru colțuri care se rotesc în sens invers acelor de ceasornic"
+    11157 "Săgeată neagră spre dreapta"
+    11160 "Vârf de săgeată echilateral în stânga, iluminat la stânga cu trei dimensiuni"
+    11161 "Vârf de săgeată echilateral în dreapta, iluminat la stânga cu trei dimensiuni"
+    11162 "Vârf de săgeată echilateral în sus, iluminat la stânga cu trei dimensiuni"
+    11163 "Vârf de săgeată echilateral în jos, iluminat la stânga cu trei dimensiuni"
+    11164 "Vârf de săgeată echilateral negru în stânga"
+    11165 "Vârf de săgeată echilateral negru în sus"
+    11166 "Vârf de săgeată echilateral negru în dreapta"
+    11167 "Vârf de săgeată echilateral negru în jos"
+    11168 "Săgeată cu vârf triunghiular în jos cu linie perpendiculară lungă spre stânga"
+    11169 "Săgeată cu vârf triunghiular în jos cu linie perpendiculară lungă spre dreapta"
+    11170 "Săgeată cu vârf triunghiular în sus cu linie perpendiculară lungă spre stânga"
+    11171 "Săgeată cu vârf triunghiular în sus cu linie perpendiculară lungă spre dreapta"
+    11172 "Săgeată cu vârf triunghiular spre stânga cu linie perpendiculară lungă în sus"
+    11173 "Săgeată cu vârf triunghiular spre dreapta cu linie perpendiculară lungă în sus"
+    11174 "Săgeată cu vârf triunghiular spre stânga cu linie perpendiculară lungă în jos"
+    11175 "Săgeată cu vârf triunghiular spre dreapta cu linie perpendiculară lungă în jos"
+    11176 "Săgeată neagră curbată în jos și în stânga"
+    11177 "Săgeată neagră curbată în jos și în dreapta"
+    11178 "Săgeată neagră curbată în sus și în stânga"
+    11179 "Săgeată neagră curbată în sus și în dreapta"
+    11180 "Săgeată neagră curbată spre stânga și în sus"
+    11181 "Săgeată neagră curbată spre dreapta și în sus"
+    11182 "Săgeată neagră curbată spre stânga și în jos"
+    11183 "Săgeată neagră curbată spre dreapta și în jos"
+    11184 "Săgeată în formă de panglică în jos și spre stânga"
+    11185 "Săgeată în formă de panglică în jos și spre dreapta"
+    11186 "Săgeată în formă de panglică în sus și spre stânga"
+    11187 "Săgeată în formă de panglică în sus și spre dreapta"
+    11188 "Săgeată în formă de panglică spre stânga sus"
+    11189 "Săgeată în formă de panglică spre dreapta sus"
+    11190 "Săgeată în formă de panglică spre stânga jos"
+    11191 "Săgeată în formă de panglică spre dreapta jos"
+    11192 "Săgeată albă în sus din bară cu bară orizontală"
+    11193 "Vârf de săgeată în sus într-o cutie dreptunghiulară"
+    11197 "Urnă cu x moale"
+    11198 "X încercuit"
+    11199 "X încercuit și greu"
```

Comment:
Please pay attention to your chosen translation for "bold" (i.e. avoid overlap with other translated terms) and be consistent.